### ISSUE TITLE
stylelint-prettier doesn't fix problems at the same time as removing empty extra lines

### ISSUE THREAD

**Issue by dmitriybo** [prettier/stylelint-prettier#354]
Here is a [minimal reproducible demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdilB3AllAC4AW8KADAA4AeA3FgL5ZYgA0IAZvgDYwBySALZxEMGsKp8AdGADOc9uAiZuAcwQgM2FOhBSArmvyY5e8gG09cogE8+PE0QC0VAE4wiRfDDd6AumxYOHpuBnxmCGjBOLogfJhqpM4AXr4QzpgZBpj4ROYoRGEwQToh+h5ePm4A9O6e3r4FRQYwMcyYjEqQqvhqAGIQbkJIRJoAVnIqSrBUioja5Tb2MI6Y+VF6AIwAbNIALNIUeqVLdg5OrpWNfpsgAKxHR3pMIIxAA):
```css
a {
  width: 0px; /* -> length-zero-no-unit problem; not autofixed */
}
  /* -> prettier/prettier problem; autofixed */
```
```js
// .stylelintrc.json
{
  "plugins": ["stylelint-prettier"],
  "rules": {
    "length-zero-no-unit": true,
    "prettier/prettier": true
  }
}
```

**What version of `stylelint`, `prettier` and `stylelint-prettier` are you using?**
```json
{
  "stylelint": "16.4.0",
  "stylelint-prettier": "5.0.0",
  "prettier": "3.2.5"
}
```

**Comment by Sergiobop** [comment]
Happens to me too, when using stylelint-prettier/recommended and stylelint-config-clean-order

I have to run stylelint twice to make it work

**Comment by nitzcard** [comment]
same

**Comment by dartess** [comment]
The problem is broader - any change in the code by the prettier prevents other errors from being fixed

**Comment by BPScott** [comment]
Thanks for the fix @dartess. I've published 5.0.1 which contains the fix.

**Comment by gezhicui** [comment]
If my project can only use CJS modules,and my usage scenario requires the use of "require" for import rather than using string declarations. but 5.x is an ESM module, what should I do? Could you release a new version 4.x to fix this issue?
